packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -196,7 +196,7 @@ abstract class HttpRequestData extends NetworkRequest {
   /// Merges the information from another [HttpRequestData] into this instance.
   void merge(HttpRequestData data);
 
-  static List<Cookie> _parseCookies(List? cookies) {
+  static List<Cookie> _parseCookies(List<dynamic>? cookies) {
```

Comment:
should this be type Cookie instead of dynamic?

Reply:
Nope. Downstream code code passes dynamic here.